packages/@sanity/vision/src/components/JsonDump.js

```diff
@@ -3,12 +3,22 @@ import React, {Fragment} from 'react'
 import PropTypes from 'prop-types'
 import ReactJson from 'react-json-view'
 
-class JsonBlock extends React.Component {
+function typeCheck(data) {
```

Comment:
For the sake of clarity maybe rename this function to `isJSONValue` (or something), and invert the check to be instead `return data !== null && typeof data === 'object'`?

If you think that's a good idea, the ternary in render below needs to be flipped too.